src/sentry/constants.py

```diff
@@ -232,6 +234,41 @@ def get_all_languages():
 
 NATIVE_UNKNOWN_STRING = '<unknown>'
 
+# to go from a marketing page slug like /for/android/ to the integration id
+# (in _platforms.json), for looking up documentation urls, etc.
+MARKETING_SLUG_TO_INTEGRATION_ID = {
+    # java
+    "java": "java",
+    "kotlin": "java",
+    "scala": "java",
+    "android": "java-android",
+
+    # TODO: add more...
+}
+
+# ???
+# via Eric: sdk name -> "server/product/event pipeline" slug
+# via Eric: python.flask -> whatever you'd send in the `integration` field in the events
```

Comment:
I don't understand this other lookup you needed. You need to go from real event data (so platform, SDK name, integrations) to... what?